src/plugins/home/server/tutorials/instructions/filebeat_instructions.ts

```diff
@@ -13,268 +13,309 @@ import { getSpaceIdForBeatsTutorial } from './get_space_id_for_beats_tutorial';
-        defaultMessage: 'Edit the configuration',
-      }),
-      textPre: i18n.translate('home.tutorials.common.filebeatInstructions.config.rpmTextPre', {
-        defaultMessage: 'Modify {path} to set the connection information:',
-        values: {
-          path: '`/etc/filebeat/filebeat.yml`',
-        },
-      }),
-      commands: [
-        'output.elasticsearch:',
-        '  hosts: ["<es_url>"]',
-        '  username: "elastic"',
-        '  password: "<password>"',
-        'setup.kibana:',
-        '  host: "<kibana_url>"',
-        getSpaceIdForBeatsTutorial(context),
-      ],
-      textPost: i18n.translate('home.tutorials.common.filebeatInstructions.config.rpmTextPost', {
-        defaultMessage:
-          'Where {passwordTemplate} is the password of the `elastic` user, {esUrlTemplate} is the URL of Elasticsearch, \
-and {kibanaUrlTemplate} is the URL of Kibana.',
-        values: {
-          passwordTemplate: '`<password>`',
-          esUrlTemplate: '`<es_url>`',
-          kibanaUrlTemplate: '`<kibana_url>`',
-        },
-      }),
-    },
-    WINDOWS: {
-      title: i18n.translate('home.tutorials.common.filebeatInstructions.config.windowsTitle', {
-        defaultMessage: 'Edit the configuration',
-      }),
-      textPre: i18n.translate('home.tutorials.common.filebeatInstructions.config.windowsTextPre', {
-        defaultMessage: 'Modify {path} to set the connection information:',
-        values: {
-          path: '`C:\\Program Files\\Filebeat\\filebeat.yml`',
-        },
-      }),
-      commands: [
-        'output.elasticsearch:',
-        '  hosts: ["<es_url>"]',
-        '  username: "elastic"',
-        '  password: "<password>"',
-        'setup.kibana:',
-        '  host: "<kibana_url>"',
-        getSpaceIdForBeatsTutorial(context),
-      ],
-      textPost: i18n.translate(
-        'home.tutorials.common.filebeatInstructions.config.windowsTextPost',
-        {
-          defaultMessage:
-            'Where {passwordTemplate} is the password of the `elastic` user, {esUrlTemplate} is the URL of Elasticsearch, \
-and {kibanaUrlTemplate} is the URL of Kibana.',
+        }),
+        commands: [
+          'output.elasticsearch:',
+          '  hosts: ["<es_url>"]',
+          '  username: "elastic"',
+          '  password: "<password>"',
+          '  ssl.ca_sha256: "<es cert fingerprint>"',
+          '  ssl.verification_mode: self_signed_ca',
+          'setup.kibana:',
+          '  host: "<kibana_url>"',
+          getSpaceIdForBeatsTutorial(context),
+        ],
+        textPost: i18n.translate(
+          'home.tutorials.common.filebeatInstructions.config.osxTextPostMarkdown',
+          {
+            defaultMessage:
+              'Where {passwordTemplate} is the password of the `elastic` user, {esUrlTemplate} is the URL of \
+              Elasticsearch, and {kibanaUrlTemplate} is the URL of Kibana. To [configure SSL]({configureSslUrl}) with the \
```

Comment:
We should mention when configuring SSL is required for the deployment, if possible. Tutorials help guide folks through the process flow and giving more information around what is required and what isn't.

Reply:
I've added a comment to the yaml block that indicates this, let me know what you think. Additionally, the docs that we're linking to here will be updated with additional information before the 8.0 release.

Reply:
@mostlyjason Feel free to let me know if you think we should make any adjustments to the copy here.